tests/Feature/Http/Controllers/BookmarkControllerTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Tests\Feature\Http\Controllers;
+
+use App\Models\Bookmark;
+use App\Models\User;
+use GroupsTableSeeder;
+use Tests\TestCase;
+
+/**
+ * @see \App\Http\Controllers\BookmarkController
+ */
+class BookmarkControllerTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->seed(GroupsTableSeeder::class);
+    }
+
+    /**
+     * @test
+     */
+    public function destroy_returns_an_ok_response()
+    {
+        $user = factory(User::class)->create();
+
+        $bookmark = factory(Bookmark::class)->create([
+            'user_id' => $user->id,
+        ]);
+
+        $response = $this->actingAs($user)->delete(route('bookmarks.destroy', ['id' => $bookmark->torrent_id]));
```

Comment:
$response = $this->actingAs($user)->delete(route('bookmarks.destroy', ['id' => $bookmark->id]));